<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video1" />
    <param name="image_width" value="800" />
    <param name="image_height" value="448" />
    <param name="pixel_format" value="yuyv" />
    <!-- <param name="color_format" value="yuv422p" /> -->
    <!-- <param name="camera_frame_id" value="usb_cam" /> -->
    <!-- <param name="io_method" value="mmap"/> -->
  </node>
  <node name="video_source_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam/image_raw"/>
  </node>
  <node name="result_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/result_frame"/>
  </node>
  
  <node name="color_tracker" pkg="cam_tracker" type="tracking.py">
  </node>
</launch>
